debian/zookeeper/Dockerfile.rpm

```diff
@@ -29,7 +29,11 @@ ENV COMPONENT=zookeeper
 
 RUN echo "===> installing ${COMPONENT}..." \
     && yum -q -y update \
-    && yum install -y confluent-kafka-${SCALA_VERSION}-${KAFKA_VERSION} \
+    && if [ "x${ALLOW_UNSIGNED}" = "xtrue" ]; then \
+        yum install -y --nogpgcheck confluent-kafka-${SCALA_VERSION}-${KAFKA_VERSION}${CONFLUENT_PLATFORM_LABEL}; \
```

Comment:
I thought I was doing this (1 for enabled) -- Apparently the unsigned repo config needs gpgcheck=0 explicitly, or it will inherit a default of 1. Confirmed that this works when setting it to 0 in the repo config.